### PR TITLE
add HTMLets to jijicagnon wiki per request

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

**Comment by Reception123** [comment]
@addshore @NDKilla

**Comment by addshore** [comment]
-2 Something went wrong, this is not loaded on extload yet

**Comment by NDKilla** [comment]
@addshore y u no gerrit?

**Comment by addshore** [comment]
?

**Comment by NDKilla** [comment]
-2/+2 is from Gerrit. Why dont we use that ;p

**Comment by addshore** [comment]
hah, yes, but people know that I mean

File: roles/mediawiki/files/localsettings/LocalExtensions.php [hunk 1]
```diff
@@ -214,6 +214,9 @@
 		$wgUploadPath = "$wgScriptPath/img_auth.php";
 		$wgUploadDirectory = '/mnt/mediawiki/private/uploads/jasperinternal.orain.org/images'; // Directory inaccessible via web access
 		break;
+	case 'jicigagnonwiki'
```
Comment (NDKilla) [inline roles/mediawiki/files/localsettings/LocalExtensions.php]: needs a ; at end of line

**Comment by NDKilla** [comment]
@addshore this extension is loaded on extload...
the build failed because it was missing a ; at the end of a line

**Comment by NDKilla** [comment]
@Reception123 I'm a collab on your Reception123/write branch. If I was a collab on your account I'd fix this ;P (btw patch-33 really? why don't you ever delete old branches xD)

**Comment by NDKilla** [comment]
This extension actually just now loaded on prod9. It was causing errors because the file didn't exist on prod9 and it was loaded on extload. I've manually run the ansible and will merge this now :)

**Comment by addshore** [comment]
@NDKilla when I looked at extload this extension was not loaded ;)
I guess since then it got updated! :)

I really need to work on sorting all of this stuff out, but, ofc, no time! :)

**Pull request merged**